pkgs/development/libraries/science/math/nccl/default.nix

```diff
@@ -17,16 +25,29 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
-  nativeBuildInputs = [ which addOpenGLRunpath ];
+  nativeBuildInputs = [
+    which
+    addOpenGLRunpath
```

Comment:
Would the `autoAddOpenGLRunpathHook` also work here, or do we need to manually invoke it in `postFixup`?

Reply:
`autoAddOpenGLRunpathHook` should work, but we better test